int16 go_near_guest_time